spec/actors/curation_concerns/file_actor_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'spec_helper'
+
+describe CurationConcerns::FileActor do
```

Comment:
TODO: Open a ticket to test that FileActor.revert_file dispatches to File#restore_version and CurationConcerns::VersioningService.create as appropriate to File#save